src/main/resources/META-INF/native-image/proxy-config.json

```diff
@@ -0,0 +1,3 @@
+[
+  ["io.lettuce.core.api.sync.RedisCommands","io.lettuce.core.cluster.api.sync.RedisClusterCommands"]
```

Comment:
We need also the following types (going through all places where we create proxies):

* AsyncNodeSelection
* HasTargetConnection
* NodeSelection
* NodeSelectionAsyncCommands
* NodeSelectionCommands
* NodeSelectionPubSubAsyncCommands
* NodeSelectionPubSubCommands
* NodeSelectionPubSubReactiveCommands
* PubSubAsyncNodeSelection
* PubSubNodeSelection
* PubSubReactiveNodeSelection
* RedisAdvancedClusterCommands
* RedisClusterAsyncCommands
* RedisClusterCommands
* RedisClusterPubSubCommands
* RedisCommands
* RedisPubSubAsyncCommands
* RedisPubSubCommands
* RedisSentinelCommands